backend/src/db-utils.js

```diff
@@ -562,20 +562,16 @@ const getLastYesterdayBlock = async (timestamp) => {
 
 // pass 'days' to set period of calculation
 const calculateFeesByDay = async (days = 1) => {
-  return await queryRows(
+  return await querySingleRow(
     [
       `SELECT
-        DATE_TRUNC('day', TO_TIMESTAMP(DIV(eo.executed_in_block_timestamp, 1000*1000*1000))) AS date,
-        SUM(transactions.receipt_conversion_tokens_burnt + eo.tokens_burnt) AS fee
-      FROM transactions
-      LEFT OUTER JOIN receipts on receipts.originated_from_transaction_hash = transactions.transaction_hash
-      JOIN execution_outcomes eo on eo.receipt_id = receipts.receipt_id
+        DATE_TRUNC('day', NOW() - INTERVAL '${days} day') AS date,
+        SUM(execution_outcomes.tokens_burnt) AS fee
+      FROM execution_outcomes
       WHERE
-        block_timestamp >= (CAST(EXTRACT(EPOCH FROM DATE_TRUNC('day', NOW() - INTERVAL '${days} day')) AS bigint) * 1000 * 1000 * 1000)
+        executed_in_block_timestamp >= (CAST(EXTRACT(EPOCH FROM DATE_TRUNC('day', NOW() - INTERVAL '${days} day')) AS bigint) * 1000 * 1000 * 1000)
       AND
-        block_timestamp < (CAST(EXTRACT(EPOCH FROM DATE_TRUNC('day', NOW())) AS bigint) * 1000 * 1000 * 1000)
-      GROUP BY date
-      ORDER BY date`,
+        executed_in_block_timestamp < (CAST(EXTRACT(EPOCH FROM DATE_TRUNC('day', NOW())) AS bigint) * 1000 * 1000 * 1000)`,
```

Comment:
I believe it should be since otherwise it will have accumulative result:

```suggestion
        executed_in_block_timestamp < (CAST(EXTRACT(EPOCH FROM DATE_TRUNC('day', NOW() - INTERVAL `${days - 1} day`)) AS bigint) * 1000 * 1000 * 1000)`,
```

Reply:
Yeah, you were right! The data by 7 days was different. But still we need to double check the data. The request performes much faster for now.

Here is responce for old request for 7 days (for 1 day it's fine):
```
[
  {
    date: "2021-08-04",
    collected_fee_in_yoctonear: "108712748090719789500000000"
  },
  {
    date: "2021-08-03",
    collected_fee_in_yoctonear: "266573976092500404200000000"
  },
  {
    date: "2021-08-02",
    collected_fee_in_yoctonear: "418585076439480818200000000"
  },
  {
    date: "2021-08-01",
    collected_fee_in_yoctonear: "596454296198668415200000000"
  },
  {
    date: "2021-07-31",
    collected_fee_in_yoctonear: "780808153074164168300000000"
  },
  {
    date: "2021-07-30",
    collected_fee_in_yoctonear: "947106645032336216200000000"
  },
  {
    date: "2021-07-29",
    collected_fee_in_yoctonear: "1118596976490000617300000000"
  }
]
```

New request:
```
[
  {
    date: "2021-08-04",
    collected_fee_in_yoctonear: "108712748090719789500000000"
  },
  {
    date: "2021-08-03",
    collected_fee_in_yoctonear: "157861228001780614700000000"
  },
  {
    date: "2021-08-02",
    collected_fee_in_yoctonear: "152011100346980414000000000"
  },
  {
    date: "2021-08-01",
    collected_fee_in_yoctonear: "177869219759187597000000000"
  },
  {
    date: "2021-07-31",
    collected_fee_in_yoctonear: "184353856875495753100000000"
  },
  {
    date: "2021-07-30",
    collected_fee_in_yoctonear: "166298491958172047900000000"
  },
  {
    date: "2021-07-29",
    collected_fee_in_yoctonear: "171490331457664401100000000"
  }
]
```

Reply:
Yeah, it was clearly the cumulative sum of the collected fees. The new values look reasonable to me.